desktop/start.js

```diff
@@ -9,25 +9,35 @@ portfinder.getPortPromise({
     port: basePort,
 }).then((port) => {
     const devServer = `webpack-dev-server --config config/webpack/webpack.dev.js --port ${port} --env platform=desktop`;
-    const buildMain = 'webpack --config config/webpack/webpack.desktop.js --config-name desktop-main --mode=development';
+    const buildMain = 'webpack watch --config config/webpack/webpack.desktop.js --config-name desktop-main --mode=development';
+
+    const env = {
+        PORT: port,
+        NODE_ENV: 'development',
+    };
 
     const processes = [
+        {
+            command: buildMain,
+            name: 'Main',
+            prefixColor: 'blue.dim',
+            env,
+        },
         {
             command: devServer,
             name: 'Renderer',
             prefixColor: 'red.dim',
+            env,
         },
         {
-            command: `${buildMain} && wait-port localhost:${port} && electron desktop/dist/main.js`,
-            name: 'Main',
+            command: `wait-port localhost:${port} && npx electronmon ./desktop/dev.js`,
```

Comment:
we can either use `npx electronmon` or install it as dev dependency and then just use `npm` 
I opted for `npx` since we already use it in package.json scripts and to have less changes for package and package-lock